cogs/Misc.py

```diff
@@ -1,36 +1,151 @@
+import csv
+import io
+import os
+import sqlite3
+from dataclasses import dataclass
+from typing import Iterable
+
 import discord
+from discord.commands.context import ApplicationContext
 from discord.ext import commands
+from discord.interactions import Interaction
+
+DATABASE_FILES = [
+    "util/database.sqlite",
+    "bandwidth.db",
```

Comment:
bandwidth.db should not be included, It's only useful for me
**You might want to add the logfile though.**

Reply:
Fixed in most recent [commit](https://github.com/Arborym/PaxAcademia/pull/135/commits/35b4794f8a72372fd8044a045ab0c96abd45473c)